src/transforms.ts

```diff
@@ -31,6 +31,8 @@ export function transformIdentity(identity: DeserializedIdentityResponse): Ident
   return {
     ...identity,
     birthDate: identity.birthDate && new Date(identity.birthDate),
+    createdAt: new Date(identity.createdAt),
```

Comment:
Do you we need to inform created and updated date params for entities like identity in the sdk? Shouldnt this be related only in the execution or the item?

Reply:
not sure if this is needed (probably not) but we are currently in the task of having the SDKs reflect the same behaviour as the API responses. If the value here is incorrect, then we should consider actually removing it from the API response itself first. I don't think there is enough merit to remove it though.
@Gabrielpanga what do you think?

Reply:
Agree with Tomi's comment. I understand the point of not having it, but for now, the idea is to reflect the API response